src/lib/API/api.js

```diff
@@ -481,6 +482,78 @@ export class APIService {
 
     return this.sharedClient.post(url, payload, options)
   }
+
+  async tatumQuery(address, chainId, from = null): Promise<any[]> {
+    const url = '/data/transactions'
+    const { CELO, MAINNET, GOERLI } = NETWORK_ID
+
+    let chain
+    const txs = []
+
+    switch (chainId) {
+      case MAINNET:
+        chain = 'ethereum'
+        break
+      case GOERLI:
+        chain = 'ethereum-goerli'
+        break
+      case CELO:
+        chain = 'celo'
+        break
+
+      // FUSE not supported on Tatum
+      default:
+        throw new Error('Chain not supported')
+    }
+
+    const params = { chain, addresses: address, transactionTypes: 'native', offset: 0 }
```

Comment:
I may be missing something, but isn't `native` only of relevance for Ethereum/Goerli? I mean on Celo, we only care about G$ (Which I believe can be monitored by the `fungible` transactionType chained with a G$ specific filter)

Additionally, I believe on Ethereum/Goerli, `internal` transaction (native balance updates caused by smart contract interactions) are also of relevance

@sirpy @johnsmith-gooddollar wdyt?

Reply:
Okay, just talked to @johnsmith-gooddollar and realized that this branch also can display native balances for Fuse and Celo, so disregard my comment. 

On Ethereum/Goerli where we don't support claiming, maybe `internal` could still be nice to have - but not necessarily necessary for now. Feel free to resolve ☺